### DISCUSSION TITLE
Error during configuring Azure AD authentication

### DISCUSSION THREAD

**Discussion by foolishhare** [requarks/wiki#4622] [Error / Bug Report]
I am created Application in Azure AD, and configure as documentation is describe.
But when I click on Apply button server throw an error.
![image](https://user-images.githubusercontent.com/14123319/138457153-8f528c14-7e74-4dcf-b201-4885b7502573.png)
But my Redirect URI is configured as documentation say.
![image](https://user-images.githubusercontent.com/14123319/138457025-8cf102ae-3a0c-412d-949b-e3243ffc182a.png)

**Comment by ghost** [comment]
Your Redirect URI is wrong. The documentation is insufficient here. Go to your AAD Authentication Strategy in your Wiki.js and scroll down. There you will find Callback URL / Redirect URI. Use this one instead. There "azure" is replaced with a number of characters.

**Reply by amal-viswa** [reply]
Hey, i'm having trouble while setting up azure ad authentication for wiki.js , I've followed the wiki.js official documentation for azure ad , but still not able to login error . can you guys help me resolve it

error 
![Screenshot 2022-08-23 at 18-12-26 Error](https://user-images.githubusercontent.com/90927394/186367982-ee356784-de37-4ebe-ace4-2c7d362d2386.png)
-

**Reply by kalletoxic** [reply]
I have the same issue as above

**Reply by deeepakkumarthakur16** [reply]
I also have that same issue when i debug little bit this is related to cookies if you try to login in incognito mode or after clearning cookies should be able to login.

**Reply by joerggollnick** [reply]
I had same issue, but solved with a random Cookie Encryption String.
